tests/e2e_operations.py

```diff
@@ -149,8 +150,77 @@ def run_configuration_file_checks(args):
         assert rc == 0, f"Failed to run tutorial script: {rc}"
 
 
+@reqs.description("Forced ledger chunk")
+def test_forced_ledger_chunk(args):
+    txs = app.LoggingTxs("user0")
+    with infra.network.network(
+        args.nodes,
+        args.binary_dir,
+        args.debug_nodes,
+        args.perf_nodes,
+        pdb=args.pdb,
+        txs=txs,
+    ) as network:
+        network.start_and_join(args)
+        primary, _ = network.find_primary()
+
+        # Submit some dummy transactions
+        network.txs.issue(network, number_txs=args.sig_tx_interval // 2)
+
+        # Submit a proposal to force a ledger chunk at the following signature
+        proposal_body, careful_vote = network.consortium.make_proposal(
+            "request_ledger_chunk", node_id=primary.node_id
+        )
+        proposal = network.consortium.get_any_active_member().propose(
+            primary, proposal_body
+        )
+        proposal = network.consortium.vote_using_majority(
+            primary,
+            proposal,
+            careful_vote,
+            wait_for_global_commit=True,
+            timeout=3,
+        )
+        assert proposal.state == ProposalState.ACCEPTED
+
+        # Issue some more transactions
+        network.txs.issue(network, number_txs=args.sig_tx_interval * 2)
+
+        ledger_dirs = primary.remote.ledger_paths()
+        network.stop_all_nodes(skip_verification=False)
```

Comment:
Stopping all the nodes in a `test_` means that the test cannot be included in the suite. Could you use a variant of `network.get_latest_ledger_public_state()` instead?

Reply:
Yes, I totally agree. The only reason I do this is because the default signature interval here is quite large, I wanted to have a small one so that it doesn't take forever.

Reply:
You can also wait for `sig_ms_interval` (which is set at 100ms for the e2e test by default) to have a new signature with an existing network.

Reply:
Right, that's an ok time to just wait for it, that should work.